types/react-resolver/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": false,
```

Comment:
For new packages, `strictFunctionTypes` should be enabled.
At least you can disable the rule by adding `// @ts-ignore` in source code: https://github.com/Microsoft/TypeScript/wiki/What%27s-new-in-TypeScript#suppress-errors-in-ts-files-using--ts-ignore-comments

Reply:
I'm getting an error in the tests that says `typeof Page` is not compatible with `ComponentType<P>` because it is not assignable to `StatelessComponent<P>`, even though `Page extends React.Component<P>`. I think this is an issue in React types.